bin/tictactoe

```diff
@@ -1,3 +1,56 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+#require "pry"
+def start
+  player_amount = ""
+  until player_amount == "0" || player_amount == "1" || player_amount == "2"
+    puts "Welcome to Tic-Tac-Toe!"
+    puts "Please select the number of players."
+    puts "0 / 1 / 2 "
+    player_amount = gets.strip
+  end
+  #binding.pry
+  player1 = " "
+  until player1 == "x" || player1 ==  "o"
```

Comment:
If someone will input any other character on the keyboard the program breaks (I tried it to be sure). That's why I specified untill == o  || == x